doc/rbd/iscsi-initiator-esx.rst

```diff
@@ -0,0 +1,33 @@
+----------------------------------
+The iSCSI Initiator for VMware ESX
+----------------------------------
+
+**Prerequisite:**
+
+-  VMware ESX 6.0 or later
+
+**iSCSI Discovery and Multipath Device Setup:**
+
+#. From vSphere, open the Storage Adapters, on the Configuration tab. Right click
+   on the iSCSI Software Adapter and select Properties.
```

Comment:
We need to set the iSCSI Recovery Timeout to 20.

Between these 2 steps, we should add a line like this:

In the General tab click the "Advanced" button and in the "Advanced Settings" set RecoveryTimeout to 20.